L1Trigger/L1THGCal/interface/backend/HGCalStage2ClusterDistribution.h

```diff
@@ -0,0 +1,35 @@
+#ifndef __L1Trigger_L1THGCal_HGCalStage2ClusterDistribution_h__
+#define __L1Trigger_L1THGCal_HGCalStage2ClusterDistribution_h__
+
+#include <array>
+#include <string>
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "L1Trigger/L1THGCal/interface/HGCalTriggerGeometryBase.h"
+
+#include "DataFormats/L1THGCal/interface/HGCalTriggerCell.h"
+#include "DataFormats/L1THGCal/interface/HGCalCluster.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "L1Trigger/L1THGCal/interface/HGCalTriggerTools.h"
+
+#include "L1Trigger/L1THGCal/interface/HGCalBackendDetId.h"
```

Comment:
It seems that many includes are not needed

Reply:
removed the unncessary ones